<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vigener</class>
 <widget class="QMainWindow" name="vigener">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Шифр Виженера</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="inLabel">
       <property name="text">
        <string>Выберите входной файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Содержимое</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="outLabel">
       <property name="text">
        <string>Выберите выходной файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Содержимое</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="keyLabel">
       <property name="text">
        <string>Введите ключ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>10</y>
      <width>411</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="openInput">
       <property name="text">
        <string>Открыть..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="openOutput">
       <property name="text">
        <string>Открыть..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataOutput"/>
     </item>
     <item>
      <widget class="QLineEdit" name="keyEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="encodeButton">
       <property name="text">
        <string>Кодировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decodeButton">
       <property name="text">
        <string>Декодировать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
